src/pages/styles.module.css

```
@@ -27,11 +27,56 @@
 .features {
   display: flex;
   align-items: center;
+  text-align: center;
   padding: 2rem 0;
   width: 100%;
 }
 
 .featureImage {
-  height: 200px;
-  width: 200px;
+  width: 100px;
+  margin-bottom: 15px;
+}
+
+.listImage {
+  max-width: 50px;
+  margin: 5px 7px 5px auto;
+  padding: 5px;
+  vertical-align: middle;
+  background-color: #ffffff;
+  border-radius: 25px;
+}
+
+.unOrderedList {
+  list-style-type: none;
+  margin:0px;
+  padding: 0px;
+}
+
+.videoContainer {
+  margin: 15px auto;
+  text-align:center;
+  max-width: 600px;
+}
+
+.textFeature {
+  margin: 15px auto;
+  max-width: 966px;
+  text-align: left;
+}
+
+.formFeature {
+  margin: 50px auto;
+  max-width: 400px;
+  text-align: left;
+}
+
+.sdkTakeMeToTutorialLogo {
+  margin: 5px 15px;
+  max-width: 80px;
+  padding:10px;
+  vertical-align: bottom;
+}
+
+.sdkTakeMeToTutorialLogo:hover {
+  opacity: 50%;
 }
```

fading things on hover is less intuitive - what about growing it slightly?


```suggestion
.sdkTakeMeToTutorialLogo {
  margin: 5px 15px;
  max-width: 80px;
  padding:10px;
  vertical-align: bottom;
  transition: transform 1s;
}

.sdkTakeMeToTutorialLogo:hover {
  transform: scale(1.1);
}
```

---

Ya, way cleaner.
Also the fading wasn't working in the Netlify preview...
This is simpler and looks nicer.